### DISCUSSION TITLE
Reverse proxy function ?

### DISCUSSION THREAD

**Discussion by Conkernel** [azukaar/Cosmos-Server#331] [Ideas]
Hi.

I'm testing your cosmos-server project. In my opinion it's a very interesting one, and I think it has most of the features I was expecting in a project like this. The only thing I miss is to be able to manage a reverse proxy, like nginx-proxy-manager or nginx-proxy. I use the first one to manage and access all my docker containers from the internet, and to get all the free certs from /Let's encrypt/ACME. I know that the certs can be also managed from cosmos, but it would be great i we could also manage all the proxy things from inside. Because of that, I still need to keep nginx-proxy-manager alive.

Is it expected to provide such a feature in a short time?

Thanks a lot, and congrats for this great project!

**Comment by azukaar** [comment]
You can manage all the URL manually in the URL tab like in NPM already

and thanks!
